### DISCUSSION TITLE
magit-merge-into and magit-merge-absorb have almost the same help text but different code

### DISCUSSION THREAD

**Discussion by landtuna** [magit/magit#4543] [Support] [answered]
https://github.com/magit/magit/blob/16316ed16d7eac48c045f5ed6a26ad437abbec93/lisp/magit-merge.el#L134

I don't understand the difference between these two functions, and the help text is almost the same.

**Comment by tarsius** [comment, marked as answer]
- If you are on `master`, then use `magit-merge-absorb` (giving `feature` as the argument) to merge `feature` into `master`.
- If you are on `feature`, then use `magit-merge-into` (giving `master` as the argument) to merge `feature` into `master`.
